### PR TITLE
driver/networkusbstoragedriver: support images configuration

### PR DESCRIPTION
**Pull request by Emantor**
# General
Implement image loading support from the environment configuration file for the NetworkUSBStorageDriver.

## Task List
- [ ] Change has been tested
- [ ] Update doc/configuration.rst
- [ ] Update CHANGES.rst

### CODE REVIEW

File: doc/configuration.rst [hunk 1]
```diff
@@ -1178,10 +1178,11 @@ Implements:
 
 .. code-block:: yaml
 
-   NetworkUSBStorageDriver: {}
+   NetworkUSBStorageDriver:
+     image: myusb.image
```
Comment (Bastian-Krause) [inline doc/configuration.rst]: This does not work. Either
`filename = self.target.env.config.resolve_path(self.image)` -> specify image filename directly
or
`filename = self.target.env.config.get_image_path(self.image)` -> specify "images subkey" and add "images" to example config.

File: doc/configuration.rst [hunk 2]
```diff
@@ -1178,10 +1178,11 @@ Implements:
...
 
 Arguments:
-  - None
+  - image (str): filename of the image to write to the remote usb storage
```
Comment (Bastian-Krause) [inline doc/configuration.rst]: See above.

File: labgrid/driver/networkusbstoragedriver.py [hunk 3]
```diff
@@ -34,7 +38,10 @@ def on_deactivate(self):
         pass
 
     @step(args=['filename'])
-    def write_image(self, filename):
+    def write_image(self, filename=None):
+        if filename is None and self.image is not None:
+            filename = self.target.env.config.get_image_path(self.image)
```
Comment (Bastian-Krause) [inline labgrid/driver/networkusbstoragedriver.py]: See above.

**Comment by Emantor** [comment]
@Bastian-Krause please take a look again

**Pull request merged**